File: java/src/main/java/demo/officeDay/generics/HealthService.java
package demo.officeDay.generics;

import demo.officeDay.linqVsStream.Person;

public class HealthService extends AbstractServiceCommunicator<Person, HealthInformation> {

    @Override
    protected Class<HealthInformation> getResponseClass() {
        return HealthInformation.class;
    }
}

File: csharp/demo.officeDay/generics/HealthService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using demo.officeDay.linqVsStream;

namespace demo.officeDay.generics
{
    public class HealthService:AbstractServiceCommunicator<Person,HealthInformation>
    {
    }
}
